integration_config.js

```diff
@@ -0,0 +1,15 @@
+const { RecruitmentServiceVerify } = require('./service_integration/recruitment_service_integration')
+
+
+const authorizationIntegrations = [RecruitmentServiceVerify]
+
+const verifyToken = async (token) => {
+    const promises = authorizationIntegrations
+        .map(integration => integration.verifyToken(token))
+    const resolvedPromises = await Promise.all(promises)
+    const result = resolvedPromises.filter(resolved => resolved !== null)
+    return result.length !== 0 ? result[0] : null
```

Comment:
Pytanie, czy tu nie chcielibyśmy wiedzieć, jaki jest też serwis, z którym się zautentykowaliśmy. Potem, jeśli już ustalimy termin spotkania, to fajnie by było, gdyby je sobie backi nawzajem chyba przesłały... choć to jest coś, co mieliśmy omówić, a tego nie zrobiliśmy.

Reply:
To fakt jest do przegadania